src/Tribe/Linked_Posts.php

```diff
@@ -1120,21 +1124,48 @@ public function saved_linked_post_dropdown( $post_type, $current = null ) {
 		// Get the label to use in placeholder attrs.
 		$label = $this->get_create_or_find_labels( $post_type, $creation_enabled );
 
+		$render_option = function ( $option ) {
```

Comment:
Wouldn't be better if we have a method for this? It could end up being more extensible.

Reply:
This seems like it needs to be a method, probably was added for quick dev and just needs to get cleaned up.

Reply:
@bordoni To be cleaned up later? Keep unresolved so you don't forget :)

Reply:
Yes it needs to be cleaned up thanks for unresolving, my mistake.